src/FakeItEasy/Creation/DefaultFakeAndDummyManager.cs

```diff
@@ -101,7 +101,9 @@ public override IFakeOptionsForWrappers<T> Wrapping(T wrappedInstance)
                 Guard.AgainstNull(wrappedInstance, nameof(wrappedInstance));
 
                 var wrapper = new FakeWrapperConfigurator<T>(this, wrappedInstance);
+#if FEATURE_SELF_INITIALIZED_FAKES
                 this.ConfigureFake(fake => wrapper.ConfigureFakeToWrap(fake));

```

Comment:
I think this statement is still necessary even without SelfInitializedFakes; it's what configures the fake to forward calls to the wrapped object. Inside `ConfigureFakeToWrap`, the call to `AddRecordingRuleWhenRecorderIsSpecified` should be disabled, though.